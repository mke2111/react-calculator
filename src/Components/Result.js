/* eslint-disable react/prop-types */
/* eslint-disable react/destructuring-assignment */
import React from 'react';
import '../index.css';

function Result(props) {
  return (
    <div>
      <h2>{props.value}</h2>
    </div>
  );
}

Result.defaultProps = {
  value: '0',
};

export default Result;
