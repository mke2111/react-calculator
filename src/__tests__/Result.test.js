import React from 'react';
import renderer from 'react-test-renderer';
import Result from '../Components/Result';

describe('Test Display component with snapshot', () => {
  it('Renders Display component correctly', () => {
    const display = renderer.create(<Result result="100" />).toJSON();

    expect(display).toMatchSnapshot();
  });
});
