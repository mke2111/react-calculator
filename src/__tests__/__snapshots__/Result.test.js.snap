// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Display component with snapshot Renders Display component correctly 1`] = `
<h2>
  100
</h2>
`;
